ci/token.sh

```diff
@@ -1,11 +1,21 @@
 #!/usr/bin/env bash
 
-cd "$(dirname "$0")/../token/js"
-
 set -e
 
-npm install
-npm run build:program
-npm run test
-npm run cluster:devnet
-npm run start
+(
+    cd "$(dirname "$0")/.."
+
+    ./do.sh update
+    ./do.sh build token
+    ./do.sh doc token
+    ./do.sh test token
+)
+
+(
+    cd "$(dirname "$0")/../token/js"
+
+    npm install
+    npm run build:program
```

Comment:
I'm not crazy about building a 2nd time.
I think if we remove this from package.json: https://github.com/solana-labs/solana-program-library/blob/f57149340401051b2e1c365410d48ea39b3eb09d/token/js/package.json#L31
we can remove this 2ndary build. Wdyt?

Reply:
We could remove the program stuff entirely from the npm flow and instead toss up a warning message if the program does not exist and force them to go through `do.sh` for al program building/updating, etc...

Reply:
I would prefer that. (I assumed since you didn't do that since last night, you found a reason to keep it)

Reply:
I'll do a follow-up pr for that, was focusing on just getting CI to work and sufficiently cover

Reply:
I'm also happy to do that, if you prefer